libraries/AP_HAL_ChibiOS/hwdef/MicoAir405v2/hwdef.dat

```diff
@@ -156,3 +156,4 @@ define HAL_FRAME_TYPE_DEFAULT 12
 
 # reduce flash usage
 include ../include/minimize_fpv_osd.inc
+undef AP_OPTICALFLOW_ENABLED
```

Comment:
I think you are better off explicitly setting the define:
```suggestion
#define AP_OPTICALFLOW_ENABLED 0
```

Reply:
The included file minimize_fpv_osd.inc define AP_OPTICALFLOW_ENABLED. And I undef it to set it default. Adding `#define AP_OPTICALFLOW_ENABLED 0` causes redefine error.